openmetadata-ui/src/main/resources/ui/src/pages/AddGlossaryTermPage/AddGlossaryTermPage.component.tsx

```diff
@@ -62,17 +62,8 @@ const AddGlossaryTermPage = () => {
     [permissions]
   );
 
-  const goToGlossaryPath = (path: string) => {
-    history.push(path);
-  };
-
-  const goToGlossary = () => {
-    const fqn = glossaryTermsFQN || glossaryName || '';
-    goToGlossaryPath(getGlossaryPath(fqn));
-  };
-
   const handleCancel = () => {
-    goToGlossary();
+    history.goBack();
```

Comment:
Are we still using this page?

Reply:
Not using anymore but haven't deleted it yet.